include/madrona/viz/viewer.hpp

```diff
@@ -80,14 +83,16 @@ class Viewer {
 
     const VizECSBridge * rendererBridge() const;
 
-    template <typename InputFn, typename StepFn>
-    void loop(InputFn &&input_fn, StepFn &&step_fn);
+    template <typename InputFn, typename StepFn, typename UIFn>
+    void loop(InputFn &&input_fn, StepFn &&step_fn, UIFn &&ui_fn);
+    int32_t getRenderedWorldID();
+    int32_t getRenderedViewID();
 
     void stopLoop();
 
 private:
     void loop(void (*input_fn)(void *, CountT, CountT, const UserInput &),
-              void *input_data, void (*step_fn)(void *), void *step_data);
+              void *input_data, void (*step_fn)(void *), void *step_data, void (*ui_fn)(void*), void* ui_data);
```

Comment:
I try to keep all Madrona's code to 80 chars wide max (I know this isn't strictly enforced but it something I maintain in 99% of the code I write). Can you wrap around these two added arguments onto a new line?